### PR TITLE
add Elm support to webpack encore

### PR DESCRIPTION
**Pull request by heithemmoumni**

### CODE REVIEW

**Comment by stof** [comment]
Is elm-webpack-loader actively maintained ? Last commit was in October 2019 and it depends on vulnerable old versions of its dependencies, which is a bad signal to me.
Encore already started the work to migrate to webpack 5 (see #645) to be ready at the time it is released. If this loader is not maintained anymore, it won't be upgraded to be compatible with webpack 5, and Encore will need to revert the feature at the time of the migration.

File: package.json [hunk 1]
```diff
@@ -33,6 +33,8 @@
     "chalk": "^4.0.0",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^3.5.2",
+    "elm": "^0.19.1-3",
+    "elm-webpack-loader": "^6.0.1",
```
[the remark this answers is not in the document]
Comment (heithemmoumni) [reply package.json:37]: yes, ok

**Comment by Rodrigo001-de** [comment]
Project made with TypeScript, react.Js, React-native. Recycling day design

https://github.com/Rodrigo001-de/Projeto-Ecoleta-advanced

File: test/WebpackConfig.js [hunk 2]
```diff
@@ -1200,12 +1200,12 @@ describe('WebpackConfig object', () => {
 
             expect(config.externals).to.deep.equals([]);
 
-            config.addExternals({ 'jquery': 'jQuery', 'react': 'react' });
+            config.addExternals({ 'jquery': 'jQuery', 'react': 'react', 'elm': 'elm' });
```
[the remark this answers is not in the document]
Comment (heithemmoumni) [reply test/WebpackConfig.js:1203]: I have followed the example, but if unnecessarily we can abort this change

[the remark this answers is not in the document]
Comment (stof) [reply test/WebpackConfig.js:1203]: This is a test about the support for configuring externals in Encore. There is no reason to make this test dependant on elm.

**Comment by stof** [comment]
@heithemmoumni any answer to my concerns about the maintenance of the elm-webpack-loader ?

File: test/WebpackConfig.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (heithemmoumni) [reply test/WebpackConfig.js:1203]: Ok

**Comment by heithemmoumni** [comment]
i already open a pull request to update all deprecated dependencies and support node 14

**Pull request closed without merging**